services/rclone.yaml

```diff
@@ -0,0 +1,35 @@
+services:
+  rclone:
+    image: nyamisty/fclone
+    entrypoint: ["rclone", "rcd", "--rc-addr", ":5572", "--rc-web-gui", "--rc-serve", "--rc-user", "rclone", "--rc-pass", "rclone", "--rc-web-gui-no-open-browser"]
+    container_name: rclone
+    restart: unless-stopped
+    expose:
+      - "5572"
+    volumes:
+      - configrclone:/config
+      - config:/globalconfig
+      - torrents:/torrents
+      - downloads:/downloads,rw,rshared
+      - backups:/backups
+      - alldata:/root/alldata
+    environment:
+      - USER_ID=${PUID}
+      - GROUP_ID=${PGID}
+      - TZ=${TZ}
+    labels:
+      - "traefik.enable=true"
+      - "traefik.http.services.rclone-seedbox.loadbalancer.server.port=5572"
+      - "traefik.http.routers.rclone.rule=Host(`rclone.${TRAEFIK_DOMAIN}`)"
+      - "traefik.http.routers.rclone.middlewares=common-auth@file, rclone-auth@docker"
+      - "traefik.http.middlewares.rclone-auth.headers.customrequestheaders.Authorization=Basic cmNsb25lOnJjbG9uZQ=="
```

Comment:
Could we avoid hardcoded sensible values?

Reply:
The Rclone webui's login feature is actually storing one's username and password, then authenticate by letting JavaScript pass the authorization header, which conflicts with traefik's authentication.
So instead, I hardcoded rclone's password, and user can use traefik's password to login rclone's webui.

Reply:
The Rclone webui's login feature is actually storing one's username and password, then authenticate by letting JavaScript pass the authorization header, which conflicts with traefik's authentication.
So instead, I hardcoded rclone's password, and user can use traefik's password to login rclone's webui.